gkShell/config/ssmtp.conf

```diff
@@ -0,0 +1,21 @@
+#
```

Comment:
Updatable file for personal configuration should be ignored in `.gitignore`

Reply:
done